torch/lib/TH/THGenerateComplexTypes.h

```diff
@@ -188,8 +187,8 @@ typedef double _Complex zx;
 
 // #pragma message "NOW DOING COMPLEX double"
 
-#define real zx
-#define accreal zx
```

Comment:
Cuda part is pretty much done, it may be that I missed a few functions. Adding re, im, arg at the moment to be able to juggle real and complex back and forth.